packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -242,6 +242,7 @@ class ValidatorsList extends React.PureComponent<Props, State> {
                 usd: weiToDecimal(+usd),
                 gold: weiToDecimal(+lockedGold),
                 elected: lastElected >= latestBlock,
+                neverElected: !lastElected,
                 online: lastOnline >= latestBlock,
                 uptime: (+score * 100) / 10 ** 24,
                 attestation:
```

Comment:
While its still unlikely that an elected validator would have 0/0 attestations, I think the best logic here would be to have the individual validator maintain `attestationsFulfilled` and `attestationsRequested` fields and then the validator group would do
SUM_ALL(attestationsFulfilled) / SUM_ALL(attestationsRequested)

This would solve for all cases, and I think it most appropriately calculates what that number represents.

If that is too difficult given the current implementation structure, what you have here should work most of the time -- it just might cause a groups score to drop dramatically when a new validator gets elected until it receives its first attestation request (once we have higher traffic this shouldn't be very long).

WDYT?

Reply:
Because of the way this data is setup, it would be a pain to calculate it that way.  But if that's the most accurate way to represent it, then I will make it work :)